src/app/api/contributor/route.ts

```diff
@@ -0,0 +1,58 @@
+import prisma from "@/lib/prisma";
+import { NextResponse, type NextRequest } from "next/server";
+
+/**
+ * one api for every single story a contributor has worked on sorted by time (given the user id)
+ * one api for the same thing, but for stories that have been staff picked
+ * I'll do do a single api that determines if we should do staff picked based on input
+ * 
+ * example url: http://localhost:3000/api/contributor?contributorId=660778a163c61d29bd4f8de4&staffPick=True
+ * Parameter: userId(id), staffPick(string: "True" | "False")
+ * return: all contributions by the userId ordered in descending order by story creation date
+ */
+
+export async function GET(req: NextRequest) {
```

Comment:
if this is for the contributor profile page, I believe Edward wanted the information search-able by contributorSlug. You could first query for the contributor doc with the matching slug if you need to use the id, maybe?